src/plugins/kibana/public/settings/sections/indices/add_data_steps/pipeline_setup/lib/selectable_array.js

```diff
@@ -0,0 +1,14 @@
+import _ from 'lodash';
+
+export default function selectableArray(array, selectedValues) {
```

Comment:
The name `selectableArray` makes it sound like this is a special class instead of a simple utility function. What about something more targeted to its actual use case, like `createCheckboxModel` or `createMultiSelectModel` or something like that?